demo-app/src/androidTestMock/java/com/worldpay/access/checkout/sample/card/NewCardConfigurationIntegrationTest.kt

```diff
@@ -0,0 +1,84 @@
+package com.worldpay.access.checkout.sample.card
+
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import com.worldpay.access.checkout.client.session.model.SessionType.CARD
+import com.worldpay.access.checkout.sample.MainActivity
+import com.worldpay.access.checkout.sample.R
+import com.worldpay.access.checkout.sample.card.CardNumberUtil.MASTERCARD_PAN
+import com.worldpay.access.checkout.sample.card.CardNumberUtil.MASTERCARD_PAN_FORMATTED
+import com.worldpay.access.checkout.sample.card.CardNumberUtil.VALID_UNKNOWN_LUHN
+import com.worldpay.access.checkout.sample.card.CardNumberUtil.VALID_UNKNOWN_LUHN_FORMATTED
+import com.worldpay.access.checkout.sample.card.standard.testutil.NewCardFragmentTestUtils
+import com.worldpay.access.checkout.sample.stub.CardConfigurationMockStub.simulateCardConfigurationServerError
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.ExternalResource
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+class NewCardConfigurationIntegrationTest {
+
+    @get:Rule
+    var cardConfigurationErrorRule = NewCardConfigurationErrorRule(ActivityScenarioRule(MainActivity::class.java))
+
+    private lateinit var cardFragmentTestUtils: NewCardFragmentTestUtils
+
+    @Before
+    fun setup() {
+        cardFragmentTestUtils = NewCardFragmentTestUtils(cardConfigurationErrorRule.innerRule)
+    }
+
+    @Test
+    fun givenCardConfigCallFails_validKnownBrandCardDetails_returnsSuccessfulResponse() {
```

Comment:
Same comment as in class above

Reply:
This method name was inherited from the class that already exists in `master`